extensions/libraries/redcore/api/payment/plugin/helper/payment.php

```diff
@@ -444,11 +444,13 @@ protected function getPaymentByExtensionOrderData($extensionName, $orderData)
 		$payment = RApiPaymentHelper::getPaymentByExtensionId($extensionName, $orderId);
 		$restrictedData = array(
 			'id', 'extension_name', 'payment_name', 'sandbox', 'created_date', 'modified_date', 'confirmed_date', 'transaction_id',
-			'amount_total', 'amount_paid', 'coupon_code', 'customer_note', 'status'
+			'amount_paid', 'coupon_code', 'customer_note', 'status'
```

Comment:
The 3 alterations here to function `getPaymentByExtensionOrderData` is to fix `amount_total` & `amount_original` being the wrong-way-around after order's total cost is changed.

https://redweb.slack.com/archives/C02951NC0/p1534236714000100
-
https://redweb.slack.com/archives/C02951NC0/p1534238084000100